internal/internal_versioning_client.go

```diff
@@ -220,6 +278,20 @@ func taskQueueTypeInfoFromResponse(response *taskqueuepb.TaskQueueTypeInfo) Task
 
 	return TaskQueueTypeInfo{
 		Pollers: pollers,
+		Stats:   statsFromResponse(response.Stats),
+	}
+}
+
+func statsFromResponse(stats *taskqueuepb.TaskQueueStats) *TaskQueueStats {
+	if stats == nil {
+		return nil
+	}
+
+	return &TaskQueueStats{
+		ApproximateBacklogCount: stats.GetApproximateBacklogCount(),
+		ApproximateBacklogAge:   stats.GetApproximateBacklogAge().AsDuration(),
+		TasksAddRate:            stats.TasksAddRate,
+		TasksDispatchRate:       stats.TasksDispatchRate,
```

Comment:
@cretz @Quinn-With-Two-Ns, realizing that eager activities are enabled by default, I'm now inclined to remove separate add/dispatch rate fields and replace both with a `BacklogIncreaseRate` field. Reason being that the individual rates can be very off an easily mislead people because they'd not include eager activities and sticky workflow tasks.
wdyt?

(The API can remain the same, or I can change that in the next release as well by deprecating the separate rates and adding a new `backlog_increase_rate` filed.)

```suggestion
	return &TaskQueueStats{
		ApproximateBacklogCount: stats.GetApproximateBacklogCount(),
		ApproximateBacklogAge:   stats.GetApproximateBacklogAge().AsDuration(),
		ApproximateBacklogIncreaseRate:            stats.TasksAddRate - stats.TasksDispatchRate,
```

Reply:
IMO we should give the same thing that the underlying API gives. If there are concerns about user confusion, they can be documented. If we must _also_ provide helpers or derived fields we can (and we should document how they are derived). The SDK should not deviate so far from the API in this point (many users will use our API directly).

Reply:
OK. I kept the separate fields with enough documentation and add a derivative field as well.